formstack-baton-requests/README.md

```diff
@@ -29,6 +42,8 @@ The UpdateDynamoHandler updates DynamoDb with submissions received by Formstack
 4. After each page of forms returned by the Formstack API have been processed, the UpdateDynamoHandler checks the remaining time the lambda has to run. If it is less than 5 minutes but there are still more forms to be processed, an UpdateDynamoResponse is returned with a `Pending` status. Otherwise, it has a `Completed` status.
 5. Once a `Completed` status is returned, the DynamoDb table storing the last run date is updated for that account.
 
+**Note:** The optional "maxUpdateSeconds" parameter should be left empty for regular SAR and RER operation to ensure the lambda does a full update. This parameter is only meant for use by devs to do partial updates during backfills or troubleshooting issues. For more details see the [FormstackUpdate Dynamo step functions](#formstackupdatedynamo) section of this document.
+
```

Comment:
I'm mostly worried by the "count" parameter that for some reason has to be initialised to the amount of forms per page in the formstack response. The problem is that this is a constant in Scala code that is not available in from the cloudformation template.
The ideal solution to this would be to remove the need to externally set this initial value or at least make it something that is more intuitive but I'm avoiding refactoring this for now